core/src/main/scala/org/bitcoins/core/wallet/builder/TxBuilder.scala

```diff
@@ -713,11 +727,46 @@ object BitcoinTxBuilder {
       feeRate: FeeUnit,
       changeSPK: ScriptPubKey,
       network: BitcoinNetwork): Future[BitcoinTxBuilder] = {
+    BitcoinTxBuilder(destinations,
+                     utxos,
+                     feeRate,
+                     changeSPK,
+                     network,
+                     lockTimeOverrideOpt = None)
+  }
+
+  def apply(
+      destinations: Seq[TransactionOutput],
+      utxos: BitcoinTxBuilder.UTXOMap,
+      feeRate: FeeUnit,
+      changeSPK: ScriptPubKey,
+      network: BitcoinNetwork,
+      lockTimeOverride: UInt32): Future[BitcoinTxBuilder] = {
```

Comment:
Maybe rather than adding a bunch of applys we just provide it as a default

```scala
lockTimeOverrideOpt: Option[UInt32] = None
```

Reply:
It isn't legal in scala to have overloaded methods define defaults as it can cause the compiler to be unable to distinguish which method is being called:
```
in object BitcoinTxBuilder, multiple overloaded alternatives of method apply define default arguments.
```